Opcodes/sfont.h

```diff
@@ -95,8 +93,9 @@ typedef struct {
 typedef struct {
         OPDS    h;
         MYFLT   *out1, *out2, *ivel, *inotnum, *xamp, *xfreq;
-        MYFLT   *instrNum, *sfBank, *iflag, *ioffset, *ienv, *iskip;
-        int32_t spltNum;
+        MYFLT   *instrNum, *sfBank, *iflag, *ioffset, *ienv;
+        int32_t
+        spltNum;
```

Comment:
Odd code formatting to have spltNum on separate line

Reply:
ok